### DISCUSSION TITLE
Expire user session

### DISCUSSION THREAD

**Discussion by keshva23** [cvat-ai/cvat#4054] [Q&A]
How can we implement session expire for user from django backend, if the user is not accessing the page for more than 5 minutes? or We can implement this feature from front end?
Thank you.

**Comment by bsekachev** [comment]
On backend you don't know if users interact with web page or not, because they can annotate one image for a long time and no any server request occurs. 

From frontend you can send logout request if no events triggered from mouse and keyboard for example.
